config/environments/development.rb

```diff
@@ -76,10 +74,6 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
Does this mean we can get rid of the listen gem? 🤞

Reply:
Good call, done :ok_hand: